shared/src/event/mod.rs

```diff
@@ -40,7 +40,7 @@ bitflags! {
     /// Flags associated with an Event entry.
     #[derive(Deserialize, Serialize)]
     #[serde(transparent)]
-    pub struct Flags: u8 {
+    pub struct Flags: u64 {
```

Comment:
Changed to avoid breaking changes in the future. It doesn't matter for serialization either way.